hardening/host-os/oscap/test.py

```diff
@@ -51,16 +51,20 @@
     # old RHEL-7 oscap mixes errors into --progress rule names without a newline
     verbose = ['--verbose', 'INFO'] if versions.oscap >= 1.3 else []
     redir = {'stderr': subprocess.STDOUT} if versions.oscap >= 1.3 else {}
+    # RHEL-7 HTML report doesn't contain OVAL findings
+    xccdf_results = '' if versions.oscap >= 1.3 else '--results results.xml'
 
     # scan the remediated system
     cmd = [
         'oscap', 'xccdf', 'eval', *verbose, '--profile', profile,
-        '--progress', '--report', 'report.html',
+        '--progress', '--report', 'report.html', *xccdf_results,
         new_ds,
     ]
     proc, lines = util.subprocess_stream(cmd, **redir)
     oscap.report_from_verbose(lines)
     if proc.returncode not in [0,2]:
         raise RuntimeError("post-reboot oscap failed unexpectedly")
 
+    reports = ['report.html', tmpdir / 'remediation.html'] if versions.oscap >= 1.3 \
+        else ['report.html', tmpdir / 'remediation.html', 'results.xml']
```

Comment:
For these longer lines, I'd personally prefer the more traditional syntax. .. Also the `>=` condition is inverted - we **want** the `results.xml` if `>= 1.3` and don't want it otherwise.
```python
if versions.oscap >= 1.3:
    logs = ['report.html', tmpdir / 'remediation.html', 'results.xml']
else:
    logs = ['report.html', tmpdir / 'remediation.html']
```
Or even
```python
logs = ['report.html', tmpdir / 'remediation.html']
if versions.oscap >= 1.3:
    logs.append('results.xml')
```